habitat-lab/habitat/config/habitat/task/objectnav_v2.yaml

```diff
@@ -0,0 +1,31 @@
+defaults:
```

Comment:
Should we add both tasks to https://github.com/facebookresearch/habitat-lab/blob/main/DATASETS.md as well?

Reply:
The task configs arent referenced on DATASETS.md file. However, the dataset configs are referenced. The dataset configs for the new episode datasets have already been referenced in that README on the main branch. They currently are pointing to a path in the main branch which doesnt exist but they should start working once this branch is merged.